http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -1299,9 +1301,10 @@ class GKEInterpreter[F[_]](
         s"helm install for RSTUDIO app ${appName.value} in project ${cluster.cloudContext.asString}"
       ).compile.lastOrError
 
+      _ <- F.sleep(1 minutes) // RStudio won't be up right away, hence wait a bit before we start ping the services
```

Comment:
1 minute could be a bit long, it could start before that..

Reply:
removed this for now